arezoo/script.js

```diff
@@ -0,0 +1,99 @@
+//1,2
+
+let i = 34;
+let j = 120;
+let z = 450;
+if (i >= 50 && i <= 99 && j >= 50 && j <= 99) {
+  console.log("True");
+} else {
+  console.log("False");
+}
+//2
+if ((i >= 50 && i <= 99) || (j >= 50 && j <= 99) || (z >= 50 && z <= 99)) {
+  console.log("True");
+} else {
+  console.log("False");
+}
+
+//3
+let a = 34;
+let b = 100;
+let c = 50;
+if (a > b && a > c) {
+  console.log("a greather than b , c");
+} else if (b > a && b > c) {
+  console.log("b greather than a , c");
+} else if (c > a && c > b) {
+  console.log("c greather than b , c");
+} else {
+  console.log("one of them is equal with another one");
+}
+//4
+let myStr = "PYwhgejwg";
+if (myStr.charAt(0) == "P" && myStr.charAt(1) == "Y") {
+  console.log(myStr);
+} else {
+  console.log("nothings");
+}
+
+//5
+let s1 = 2334;
+let s2 = 343;
+let sum = s1 + s2;
+if (sum >= 50 && sum <= 80) {
+  console.log("65");
+} else {
+  console.log("80");
+}
+
+//6
```

Comment:
//6. Check whether the sum of two integers is 8 or whether their difference is 8.

we should also check the difference is 8